### ISSUE TITLE
Focus not moved properly when clicking text in preview

### ISSUE THREAD

**Issue by laxsjo** [typst/webapp-issues#290]
### Description

# The Issue
When clicking on text in the preview window, moving the cursor to the corresponding position in the source code, the focus is not move over to the editor panel entirely. When moving the cursor left/right using the arrow keys, the preview panel scrolls up/down, as if focus was still on said panel. Note that moving the cursor up or down does _not_ affect the preview panel. This behaviour can be negated by clicking on any part of the code in the editor panel after clicking on any text in the preview, which seems to correctly move focus over to the editor entirely, making the preview scrolling unaffected by cursor movements.

![typst-bug-example](https://github.com/typst/webapp-issues/assets/72399173/112d02ac-3fd1-4774-92c8-871650ebb292)

# Reproduction Steps
1. Open any document in the Typst web app.
2. Click on any text in the preview, moving the cursor to the corresponding location in the editor.
3. Move the cursor left or right using the arrow keys.

The preview should then scroll up and down as the left and right arrow keys are pressed.

### Browsers

Firefox, Chrome, Opera

### OS

Windows

**Comment by laxsjo** [comment]
And one second after submitting I realize that I had not checked for previous duplicate issues. 🤦
This was of course already reported by https://github.com/typst/webapp-issues/issues/285.
This issue does go into more detail, but I guess I should close this?

**Comment by laurmaedje** [comment]
Yeah, I'll close this then. Feel free to leave a comment on the other issue with any additional useful details.
